instance/models/instance.py

```diff
@@ -455,10 +455,8 @@ class AnsibleInstanceMixin(models.Model):
     """
     An instance that relies on Ansible to deploy its services
     """
-    ansible_source_repo_url = models.URLField(max_length=256,
-                                              default='https://github.com/edx/configuration.git')
-
-    configuration_version = models.CharField(max_length=50, default='master')
+    ansible_source_repo_url = models.URLField(max_length=256, blank=True)
+    configuration_version = models.CharField(max_length=50, blank=True)
```

Comment:
@antoviaque Unfortunately, at the moment it's not defined what these fields are supposed to mean.  When I added code to freeze the Ansible settings at provisioning time, you had concerns that now the field with the Ansible settings doesn't reflect the configuration of the _next_ provisioning anymore. This time, you have concerns that changing a field after an instance has been deployed revises history.  In some way, this seems to be the opposite of what you said last time. :)

I think we have to redefine what an `OpenEdxInstance` is, philosophically.  Currently, it's a template that can be used to deploy a set of Open edX instances to an OpenStack cluster, but it's representing the deployed instances as well.  This mixup is also showing in the UI, e.g. if you redeploy an instance, the log for the new provisioning run is simply appended to all existing log entries, making it a bit hard to find where the current provisioning log starts.  We probably need to separate the two mentioned conepts a little better to get clearer notions here.  If we want to keep the history of provisionings correctly, we have to freeze the whole configuration everytime a provisioning run is started, and keep that.  On the other hand, the template for new provisioning runs is allowed to dynamically follow the current settings on the PR – and all global default values, if nothing was set explicitly for the instance.

To me, the default values felt out of place here, since they were the only things that were specific to Open edX on this whole mixin class.